config-connector/solutions/projects/kpt/project-services/namespace.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
```

Comment:
We also want to change the filename to 0-namespace.yaml to ensure that the namespace creation always happens the first.

Reply:
Done.